src/utils.ts

```diff
@@ -77,6 +77,13 @@ export function adaptUrl(url: string): string {
 
     return url;
 }
+/**
+ * Check if docker is Docker Desktop
+ */
+export function isDockerDesktop(): boolean {
+    const res = spawnSync("docker", ["system", "info"], { encoding: "utf8" });
+    return res?.stdout?.includes("Operating System: Docker Desktop");
```

Comment:
Why the question marks? Shouldn't these properties always exist? When I earlier said that this looks flaky, I meant that we are depending on the output format of the `docker system info command`, which might change or not be the same on all platforms.

Reply:
this morning i had to add question marks to make tests pass while it was ok yesterday :|

Reply:
yeah :) but did u know https://www.bbc.com/future/article/20221011-how-space-weather-causes-computer-errors XD

Reply:
that's it, I no longer need excuses when I can blame the universe xD